.gitignore

```diff
@@ -31,3 +31,5 @@ logs
 *data
 *.swp
 out/
+lib/
```

Comment:
Foundry's libraries are resolved as git submodule. It's better to keep the folder.